GOVERNANCE.md

```diff
@@ -177,6 +177,6 @@ The TSC follows a [Consensus Seeking][] decision-making model per the
 
 [Consensus Seeking]: https://en.wikipedia.org/wiki/Consensus-seeking_decision-making
 [TSC Charter]: https://github.com/nodejs/TSC/blob/HEAD/TSC-Charter.md
-[collaborators discussion page]: https://github.com/orgs/nodejs/teams/collaborators/discussions
+[collaborators discussion page]: https://github.com/nodejs/collaborators/discussions/new?category=collaborator-nominations
```

Comment:
```suggestion
[collaborators discussion page]: https://github.com/nodejs/collaborators/discussions
```
It's better to just link to the discussions instead of the page to create a new discussion (though for some reason that link doesn't even link to the discussion creation page although it's the correct link).

Reply:
Maybe this one then?
https://github.com/nodejs/collaborators/discussions/categories/collaborator-nominations

Reply:
Well collaborator discussions don't just include collaborator nominations so I think it would be better to link to all discussions instead of a specific category?

Reply:
But this link is specifically used in the section about collaborator nominations:
https://github.com/nodejs/node/blob/f08655532b5bf06549000f5f7dd3c833c06b62c4/GOVERNANCE.md?plain=1#L157-L161

Reply:
Oh I see, I didn't notice that part, then yeah the second link you proposed is the correct one then.